CONVENTIONS.md

```diff
@@ -546,3 +546,17 @@ used SCC there would be a delay before it could start. In recent versions of OCP
 (4.6+) this delay has been virtually eliminated, the usage of runlevels should
 not be required at all hence the primary alternative is to simply try the
 workload without any runlevel specified.
+
+#### Metrics
+
+Core operators deployed by the CVO as well as their operands are encouraged to expose metrics using the [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/). Metrics are useful for alerting and telemetry. Operators can deploy [service monitors](https://docs.openshift.com/container-platform/latest/rest_api/monitoring_apis/servicemonitor-monitoring-coreos-com-v1.html) to have their metrics collected by the in-cluster Prometheus pods running in the `openshift-monitoring` namespace.
```

Comment:
I'm ok starting with encouraged.   If we're close to complete, updating to "must" is a future possibility.

Reply:
I've udpated the text to say "should" instead of "must". Is it what you had in mind @deads2k?